wwwroot/cgi-bin/plugins/geoip.pm

```diff
@@ -63,7 +63,7 @@ sub Init_geoip {
 	# ENTER HERE CODE TO DO INIT PLUGIN ACTIONS
 	debug(" Plugin $PluginName: InitParams=$InitParams",1);
    	my ($mode,$tmpdatafile)=split(/\s+/,$InitParams,2);
-   	my ($datafile,$override)=split(/\+/,$tmpdatafile,2);
+   	my ($datafile,$override)=split(/\s+/,$tmpdatafile,2);
```

Comment:
According to doc, the separator between the datafile name ofgeoip and the option override is +
```
# PLUGIN: GeoIP_ASN_Maxmind
# REQUIRED MODULES: Geo::IP or Geo::IP::PurePerl (from Maxmind)
# PARAMETERS: [GEOIP_STANDARD | GEOIP_MEMORY_CACHE] [/pathto/GeoIPASN.dat[+/pathto/override.txt][+http://linktoASlookup]]
# DESCRIPTION: This plugin adds a chart of AS numbers where the host IP address is registered.
# This plugin can display some ISP information if included in the database. You can also provide
# a link that will be used to lookup additional registration data. Put the link at the end of
# the parameter string and the report page will include the link with the full AS number at the end.
# Replace spaces in the path of geoip data file with string "%20".
#
#LoadPlugin="geoip_asn_maxmind GEOIP_STANDARD /usr/local/geoip.dat+http://enc.com.au/itools/autnum.php?asn="

```